README.dev.md

```diff
@@ -6,11 +6,612 @@ You will need a machine with at least 2GB of RAM available (see
 https://github.com/zulip/zulip/issues/32 for a plan for how to
 dramatically reduce this requirement).
 
-Start by cloning this repository: `git clone https://github.com/zulip/zulip.git`
+Contents:
+
+* [First-time contributors start here](#first-time-contributors-start-here)
+* [Using Vagrant](#using-vagrant)
+* [Using provision.py without Vagrant](#using-provisionpy-without-vagrant)
```

Comment:
This probably should be renamed to "Installing on Ubuntu Trusty without Vagrant."

(Should be able to add Xenial to the supported list here soon).